src/cli/commands/config.ts

```
@@ -0,0 +1,511 @@
+  const { input, select, Separator } = await import('@inquirer/prompts');
+
+  const stepReg = new StepRegistry(ctx.stepsDir);
+  const availableSteps = stepReg.list().filter((s) => s.stageCategory === stageCategory);
+  const flavorSteps: { stepName: string; stepType: string }[] = [...existing];
+
+  console.log(`\nPick steps for this flavor from the "${stageCategory}" stage library:`);
+
+  while (true) {
+    if (flavorSteps.length > 0) {
+      console.log('\nSteps so far:');
+      flavorSteps.forEach((s, i) => console.log(`  ${i + 1}. ${s.stepName} (→ ${s.stepType})`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)